tailwind.config.cjs

```diff
@@ -0,0 +1,19 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+	content: ["./*.{html, js}"],
```

Comment:
Add ts to the list `{html,js}` when you transfer the work to ts

Reply:
Will do